### PR TITLE
Update NucleiSegmentationBoundaryModel

### PR DESCRIPTION
**Pull request by bioimageiobot**
This is an automatic PR created by the @bioimageiobot regarding changes to the resource item 10.5281/zenodo.5764892.
The following version(s) will be added:
  - 10.5281/zenodo.6028056

You can find their corresponding RDF files here:
  - https://zenodo.org/api/files/50a4c576-db4c-49a6-a691-59f9d96e6d5f/rdf.yaml

Please review the changes and make sure the new item or version(s) pass the following check list:
- [ ] Passed the BioImage.IO Spec validator (static validation);
- [ ] Passed the BioImage.IO CI tests (dynamic validations);
- [ ] The meta information for the RDF item is complete;
  - [ ] The tags is complete and describes the model;
  - [ ] Naming is intuitive and descriptive, example:  Multi-Organ Nucleus Segmentation (StarDist 2D);
  - [ ] Authors are provided;
  - [ ] Documentation is complete;
    * For models, include an overview, describe how the model is trained, what is the training data, how to use the model, how to validate the results and list the references. TODO: Model documentation template.
- [ ] Approved by at least one of the BioImage.IO admin team member.

Maintainers: @constantinpape

Note: If you updated or re-uploaded another version for the current item on Zenodo, this PR won't be changed automatically. To proceed, you can do the following:
 1. Block this version, but keep looking for future versions: Edit the current resource.yaml and keep the top-level `status` field as `accepted`, but change the `status` under the current version to `blocked`.
 2. Accept this version and keep looking for future versions: Merge this PR for now.
 3. Keep proposed version(s) (and this resource in general if it is new) as pending: Close this PR without merging.

 Then wait for the CI on the main branch to complete. It should detect the new version(s) and create another PR for the new version(s).

Previous PRs of this resource: none

### CODE REVIEW

**Comment by FynnBe** [comment]
@constantinpape please also check the equivalent of this PR on my fork: https://github.com/FynnBe/collection-bioimage-io/pull/11

**Comment by constantinpape** [comment]
@FynnBe why is this not running validation here?

**Comment by FynnBe** [comment]
> @FynnBe why is this not running validation here?

🐛 🐛 🐛... what do you mean? it works! 🥇

**Comment by FynnBe** [comment]
same known issue:
```yaml
channels: [pytorch, conda-forge]
dependencies: pytorch>=1.6,<2.0  # not a list
name: 10.5281zenodo.5764893

```

**Comment by constantinpape** [comment]
Then something is wrong here, it's correct on zenodo now, see "environment.yaml" in https://zenodo.org/record/6028056#.YgQ7s_so8UE.

**Comment by constantinpape** [comment]
> ```yaml
> channels: [pytorch, conda-forge]
> dependencies: pytorch>=1.6,<2.0  # not a list
> name: 10.5281zenodo.5764893
> ```

Note that this is also the old DOI, the new DOI is 10.5281/zenodo.6028056

**Comment by FynnBe** [comment]
> Note that this is also the old DOI, the new DOI is 10.5281/zenodo.6028056

Ah I see.. yes it's correct now, but this PR validates all resource versions (including the old one), which it should not... 🐛

**Comment by FynnBe** [comment]
> > Note that this is also the old DOI, the new DOI is 10.5281/zenodo.6028056
> 
> Ah I see.. yes it's correct now, but this PR validates all resource versions (including the old one), which it should not... 🐛

well, the tricky bit is that this PR does change the resource.yaml in collection/... which could potentially also change the update to the older versions (changes in resource.yaml are merely detected via sha256 and ignore where a change occurs).  I suppose we have to change that and safe the resource.yaml instead of the sha256 and compare for resource and versions separately... 
any other idea how to solve this?

File: collection/10.5281/zenodo.5764892/resource.yaml [hunk 1]
```diff
@@ -1,9 +1,21 @@
 doi: 10.5281/zenodo.5764892
 id: 10.5281/zenodo.5764892
-owners: [77626]
```
Comment (FynnBe) [inline collection/10.5281/zenodo.5764892/resource.yaml:3]: these formatting changes are an artifact of developing this repo and should not occur in the future / after a reset of gh-pages...

**Comment by FynnBe** [comment]
we could take the version info out of the resource.yaml and keep resource.yaml at collection/<resource id> and add the versions at collection/<resource id>/<version id>/version.yaml
This somewhat unnecessarily yet again increases the file number we deploy to gh-pages (shouldn't be a big problem?!?), but otherwise it makes it a bit more straight forward to handle changes and what these changes imply...

**Comment by FynnBe** [comment]
@oeway any thoughts on this?

**Comment by FynnBe** [comment]
.. I decided to keep track of the commit the main ci ran on last and compare to the collection at that time.
keeping track of that commit seems the most straight forward with a dedicated git tag

**Pull request closed without merging**